src/main/kotlin/entry/dsm/equusrecruitment/domain/application/domain/enum/Major.kt

```diff
@@ -0,0 +1,6 @@
+package entry.dsm.equusrecruitment.domain.application.domain.enum
+
+enum class Major(val value: String) {
```

Comment:
```suggestion
enum class Major {
BACKEND, FRONTEND 
}
```
이렇게 하는 방법도 있는데 굳이 생성자를 통해 값을 전달하는 이유가 뭔가요?

Reply:
클래스의 상수가 무엇인지 값을 전달하기 위해 value라는 필드를 통하여 한국어 값을 할당하였습니다. 추후에 혹시 필요할 수도 있다고 생각하여서 미리 명시해주었습니다.